lib/site_prism/element_container.rb

```diff
@@ -131,6 +133,18 @@ def create_waiter(element_name, *find_args)
       end
     end
 
+    def create_disappearance_waiter(element_name, *find_args)
+      method_name = "wait_for_#{element_name}_disappearing"
```

Comment:
@alvy54 I think wait_for_#{element_name}_disappears sounds clearer. What do you think?